CHANGES.md

```diff
@@ -19,6 +46,14 @@
     pub fn add_rule(&mut self, (name, name_span): (String, Span), actiont: Option<String>) {
   ```
 
+* `YaccParserError` has been renamed to `YaccGrammarError` and changed from an
+  `enum` (which frequently changed, breaking semver) to a `struct`. The `kind`
+  field is now internal (so won't break semver as often).
```

Comment:
There is also some removals associated with this which should probably be noted: 
Perhaps something to the effect of:

 `GrammarValidationError` and `YaccParserError` have been combined as `YaccGrammarError`, replacing the enum `YaccGrammarError` which had a variant for each of those.

Reply:
Fixed in 0b1d465.